antora.yml

```diff
@@ -4,8 +4,9 @@ version: 2021.1
 asciidoc:
   attributes:
     varVersion: 2021.1
+    bonitaTechnicalVersion: '7.12.1'
```

Comment:
Do we need those single quotes?

Reply:
Adding single quotes forces the value to be considered as a string, otherwise magic happens and it can be considered as a number.
We add issues in the past with version finishing by zero, like `7.10` that were considered as `7.1`. Here, we could remove the quotes because the value currently doesn't end with a zero.
But remember that the change will be propagated to 2021.2; so, we soon will set the value to `7.13.0` in this branch and we may have the same issue (except if we are sure 7.13.0 will be considered as a string).
So I had the single quote to prevent any problem in the 2021.2 branch.